recipes/skaware/execline.inc

```diff
@@ -39,3 +40,5 @@ LIBRARY_VERSION = "2"
 DEPENDS_${PN}-libexecline += "libc libskarnet"
 RDEPENDS_${PN}-libexecline += "libc libskarnet"
 FILES_${PN}-libexecline-dev += "${includedir}"
+
+CFLAGS += "-g -fstack-protector"
```

Comment:
Do you have an idea for why Laurent is using `-fno-stack-protector`?

Reply:
No, but I suppose his code might be penalized quite heavily. Maybe he can be convinced to make it a configure option instead of hard-coding it to no.

Reply:
I think it would be best to take a discussion with him on this before throwing the switch.  If he has good reasons for it, we should not undo it without knowing his reasons and taken that into consideration.

Reply:
Yes, I was mostly doing this to get debug symbols for valgrind, but thought I'd try throwing in stack-protector as well just to see if that might fire something; but then I found that it's a bit random whether that ends up taking effect (because in some of the software, he adds -fno-stackprotector to CPPFLAGS which precedes CFLAGS in the compile command, in others, it is appended to CFLAGS itself, and there's no other convenient variable to append to that is guaranteed to have precedence).